bin/pit-init.sh

```diff
@@ -56,21 +57,25 @@ if [ -e /dev/disk/by-label/PITDATA ]; then
     mount -v -L PITDATA
 fi
 
-PREP_DIR=/var/www/ephemeral/prep
-CSI_CONF=$PREP_DIR/system_config.yaml
+PREP_DIR=${PITDATA}/prep
+DATA_DIR=${PITDATA}/data
+CONF_DIR=${PITDATA}/configs
+[ ! -d $PREP_DIR ] && mkdir -pv $PREP_DIR
+[ ! -d $DATA_DIR ] && mkdir -pv $DATA_DIR
+[ ! -d $CONF_DIR ] && mkdir -pv $CONF_DIR
+
+
 set -e
-if [ ! -d $PREP_DIR ] ; then
-    echo >&2 "Need $PREP_DIR; this needs to be created by hand if this is a first-time install."
-    echo >&2 "See: https://github.com/Cray-HPE/docs-csm/blob/main/install/bootstrap_livecd_usb.md#configuration-payload"
-    exit 1
-elif [ ! -f $CSI_CONF ] ; then
+CSI_CONF=$PREP_DIR/system_config.yaml
+if [ ! -f $CSI_CONF ] ; then
+    # TODO: MTL-1695 Update this to point to the new example system_config.yaml file.
     echo >&2 'CSI needs inputs; no $CSI_CONF file detected!'
-    echo >&2 'See: https://github.com/Cray-HPE/docs-csm/blob/main/install/bootstrap_livecd_usb.md#generate-installation-files'
+    echo >&2 'See: https://github.com/Cray-HPE/docs-csm/blob/main/install/prepare_configuration_payload.md'
     exit 1
 fi
 echo 'Generating Configuration ...'
 (
-    pushd /var/www/ephemeral/prep
+    pushd $PREP_DIR
     rm -rf $SYSTEM_NAME && csi config init
```

Comment:
Should this be a move to back-up instead of a remove in case things go wrong?

Reply:
How about:
```bash
    pushd $PREP_DIR
    [ -d $SYSTEM_NAME ] && mv $SYSTEM_NAME $SYSTEM_NAME-$(date '+%Y%m%d%H%M%S')
    csi config init
```

Reply:
That works for me